iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -496,14 +496,6 @@ static void getUsedValuesDefinedAboveAfterCloningOps(
       valuesDefinedAbove.insert(outsideValue);
       continue;
     }
-    // Only clone if operation either has no operands, or the operation is in
-    // same basic block as the dispatch op. This could really be relaxed, but
-    // this is conservative for now.
-    if (definingOp->getNumOperands() != 0 &&
```

Comment:
I now remember why I did this. This might end up with weird use-def ordering issues if we just inline always across blocks. (We have a separate internal issue with respect to the inlining operations with `flow.ex.stream.fragment`). We should probably use dominance information to decide when this is OK. Removing this might cause issues.

In any case, this should have still created a dispatch region for the subtensor. Less efficient, but still correct. Let me see why this didnt happen